### DISCUSSION TITLE
Where are the UI tests for vscode?

### DISCUSSION THREAD

**Discussion by code2graph** [microsoft/vscode-discussions#765] [Extension Development QnA]
Where are the UI tests for vscode written using puppeteer?

**Comment by HatemMn** [comment]
can you please elaborate ?
who developed them ?

**Comment by olguzzar** [comment]
Hi @code2graph, this forum is specifically for questions regarding extension development. Based on the information provided, I'm not sure that that's what you are asking about, so I'm going to close this discussion. Please feel free to follow up if you have additional information!
